### DISCUSSION TITLE
Github education for non profit research organization (German registered association)?

### DISCUSSION THREAD

**Discussion by Siedlerchr** [community/community#22255] [New to GitHub]
<p>Hi,</p>
<p>I am asking here as I could not find an answer in the help section regarding this special case.</p>
<p>I am working for a registered <a href="https://safetyinnovation.center/?lang=en" rel="nofollow noopener">non profit research organization in Germany.</a> We are an officially registered non-profit association according to German law (eingetragener Verein, e.V.) and we would like to use Github for hosting code of our research projects. Our research topic is <em>Digitisation for civil security</em>. In addtion, we have a cooperation agreement with the University of Paderborn.</p>
<p>Therefore we would like to create a Github organization.</p>
<p>We would like to know, if we as organization are eligible for the education “discount” or mode and if yes what kind of documents do you need from us?</p>
<p>Thanks in advance</p>
<p>Regards from Germany</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/siedlerchr">@siedlerchr</a>,</p>
<p>Thanks for reaching out! </p>
<ul>
<li>
</li>
</ul>
<p>Our nonprofit program offers a free <a href="https://github.com/pricing" rel="nofollow noopener">Team Plan</a>(unlimited users and unlimited private repositories) to official nonprofit organizations and charities. We just need a little more information from you to get you set up.</p>
<p>You can read more about non-profits on GitHub, including what kinds of information we need, here:</p>
<p><a href="https://github.com/nonprofit" rel="nofollow noopener">https://github.com/nonprofit</a></p>
<p>When you’re ready to apply, please email <strong><a href="mailto:support@github.com">support@github.com</a></strong> with the following information:</p>
<p>* Your nonprofit’s GitHub organization account name<br>
* Your nonprofit’s GitHub organization URL<br>
* Proof of your nonprofit status with your local government (501©(3) status in the US)<br>
* The registered name of your nonprofit<br>
* A summary of what your nonprofit organization does<br>
* Confirmation that your organization is nongovernment, nonacademic, non-commercial, non-political and has no religious affiliation</p>
<p>I hope this helps!</p>
